File: Leetcode/2016/2016.cpp
class Solution {
public:
    int maximumDifference(vector<int>& nums) {
        int n = (int)nums.size();
        int ans = -1;
        for (int i = 0; i < n-1; ++i) {
            for (int j = i+1; j < n; ++j) {
                if (nums[i] < nums[j]) {
                    ans = max(ans, nums[j]-nums[i]);
                }
            }
        }
        return ans;
    }
};

File: Leetcode/2016/2016.go
func maximumDifference(nums []int) int {
	ans := -1
	for i := 0; i < len(nums)-1; i++ {
		for j := i+1; j < len(nums); j++ {
			if nums[i] < nums[j] {
				temp := nums[j] - nums[i]
				if temp > ans {
					ans = temp
				}
			}
		}
	}
	return ans
}